### DISCUSSION TITLE
trying to install on 7583

### DISCUSSION THREAD

**Discussion by MrDinoRawrrz** [Freetz-NG/freetz-ng#701] [Allgemein] [answered]
Hello,

I'm trying to install Freetz-NG on a fritzbox 7583. i've downloaded the linux version that was requested to have because the prerequisites page recently broke, and i did not have a linux server ready before that. so far so good, pulling the github, puting umask 0022 ,and running make menuinstaller.  that all works. but when i try to "make" i get the following error 
![image](https://user-images.githubusercontent.com/17454045/212554898-a4713372-46c6-45fa-b010-76b85de92195.png)
after i run make again (as it says, please re-run) i get a bunch of errors. and can't find out why this is happening, i have no idea where to search for a log and make | > info.txt didn't work either.
after i updated freetz-ng today again after github updates, some of the errors are gone, but i keep having some issues. eg:
![image](https://user-images.githubusercontent.com/17454045/212555668-45256b02-2d36-47cd-a022-a137b88256e5.png)

![image](https://user-images.githubusercontent.com/17454045/212555053-b2bad3c1-dd87-45d6-ab86-30172b142c75.png)

**Comment by fda77** [comment, marked as answer]
There are no errors on your screenshot, only warnings. They are called warning because they are not an error
screenshot 1: squashfs
screenshot 2: prelink
You could use `fmake` for log saving https://github.com/Freetz-NG/freetz-ng/blob/master/fmake

**Reply by MrDinoRawrrz** [reply]
Hey, Thanks, due to fmake i found out that i've been missing a package. i've installed that one and now everything works. thanks for the info.
